src/Jackett.Common/Indexers/Anidex.cs

```diff
@@ -0,0 +1,387 @@
+            AddCategoryMapping(10, TorznabCatType.TVAnime, "♫ - Lossless");
+            AddCategoryMapping(11, TorznabCatType.TVAnime, "♫ - Video");
+            AddCategoryMapping(12, TorznabCatType.TVAnime, "Games");
+            AddCategoryMapping(13, TorznabCatType.TVAnime, "Applications");
+            AddCategoryMapping(14, TorznabCatType.TVAnime, "Pictures");
+            AddCategoryMapping(15, TorznabCatType.TVAnime, "Adult Video");
+            AddCategoryMapping(16, TorznabCatType.TVAnime, "Other");
+
+            // Configure the language select option
+            SelectItem languageSelect = new SelectItem(new Dictionary<string, string>()
+            {
+                {"1", "English"},
+                {"2", "Japanese"},
+                {"3", "Polish"},
+                {"4", "Serbo-Croatian" },
+                {"5", "Dutch"},
+                {"6", "Italian"},
+                {"7", "Russian"},
+                {"8", "German"},
+                {"9", "Hungarian"},
+                {"10", "French"},
+                {"11", "Finnish"},
+                {"12", "Vietnamese"},
+                {"13", "Greek"},
+                {"14", "Bulgarian"},
+                {"15", "Spanish (Spain)" },
+                {"16", "Portuguese (Brazil)" },
+                {"17", "Portuguese (Portugal)" },
+                {"18", "Swedish"},
+                {"19", "Arabic"},
+                {"20", "Danish"},
+                {"21", "Chinese (Simplified)" },
+                {"22", "Bengali"},
+                {"23", "Romanian"},
+                {"24", "Czech"},
+                {"25", "Mongolian"},
+                {"26", "Turkish"},
+                {"27", "Indonesian"},
+                {"28", "Korean"},
+                {"29", "Spanish (LATAM)" },
+                {"30", "Persian"},
+                {"31", "Malaysian"}
+            }) { Name = "Language", Value = "1" };
+            configData.AddDynamic("languageid", languageSelect);
+
+            // Configure the sort selects
+            SelectItem sortBySelect = new SelectItem(new Dictionary<string, string>()
+            {
+                {"upload_timestamp", "created"},
+                {"seeders", "seeders"},
+                {"size", "size"},
+                {"filename", "title"}
+            }) { Name = "Sort by", Value = "upload_timestamp" };
+            configData.AddDynamic("sortrequestedfromsite", sortBySelect);
+
+            SelectItem orderSelect = new SelectItem(new Dictionary<string, string>()
+            {
+                {"desc", "Descending"},
+                {"asc", "Ascending"}
+            })
+            { Name = "Order", Value = "desc" };
+            configData.AddDynamic("orderrequestedfromsite", orderSelect);
+        }
+
+        public string SortBy
+        {
+            get
+            {
+                return ((SelectItem)this.configData.GetDynamic("sortrequestedfromsite")).Value;
+            }
```

Comment:
Remove `this` qualifier and convert to read only expression body for each of these:

ex.
`SortBy => ((SelectItem)configData.GetGetDynamic("sortrequestedfromsite")).Value;`